codebuild/common-posix.sh

```diff
@@ -6,5 +6,10 @@ mkdir /tmp/build
 cd /tmp/build
 
 cmake -DPERFORM_HEADER_CHECK=ON -DENABLE_SANITIZERS=ON $@ $CODEBUILD_SRC_DIR
-make
+make -j 12
 ctest . --output-on-failure
+rm -rf ./*
+cmake -DPERFORM_HEADER_CHECK=ON -DENABLE_SANITIZER=ON -DBUILD_SHARED_LIBS=ON $@ $CODEBUILD_SRC_DIR
```

Comment:
trivial: you could avoid the copy-paste by declaring the common stuff as a variable

```
CMAKE_ARGS="-DCMAKE_BUILD_TYPE=RelWithDebInfo -DPERFORM_HEADER_CHECK=ON -DENABLE_SANITIZERS=ON $@"

cmake $CMAKE_ARGS -DBUILD_SHARED_LIBS=ON ../
```